backend/api/v1/project_service.go

```diff
@@ -1813,9 +1804,6 @@ func (s *ProjectService) ListSchemaGroups(ctx context.Context, request *v1pb.Lis
 	if project == nil {
 		return nil, status.Errorf(codes.NotFound, "project %q not found", projectResourceID)
 	}
-	if project.Deleted {
```

Comment:
Soft-deleted resources should not be returned in List (AIP-132) responses by default (unless bool show_deleted is true).
If the parent(project) had been deleted, we should returning the resources under the parent in List children request?

Reply:
I have the same question. LGTM otherwise

Reply:
Regarding sub-resources, the API style guide doesn't specify the right behavior. We can treat get/list() of sub-resources similar to get() fields of the resource.